### DISCUSSION TITLE
Avatar provider

### DISCUSSION THREAD

**Discussion by MarJose123** [filamentphp/filament#3990] [Help] [answered]
https://github.com/filamentphp/filament/blob/b6259b6899d663239350db892c6ae31a9377e88b/packages/admin/src/AvatarProviders/UiAvatarsProvider.php#L19

Is there a way to use other Avatar provider like https://github.com/laravolt/avatar as this working for offline/intranet web system

**Comment by danharrin** [comment, marked as answer]
Yes, you can swap out the `default_avatar_provider` class with your own in config
